change/@ni-nimble-components-278f4b89-b1f8-402d-a3e7-ba0338b1b348.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
There are some possibly unexpected diffs in the latest Chromatic build:
1. select components in a lot of stories got narrower
2. the listbox component is no longer rendering any items

Reply:
1. Yep, this is intentional
2. We've talked through this off-line and we will be deleting the listbox component as nothing is using it.